Carmunication/Views/SessionStart/GroupCollectionViewCell.swift

```diff
@@ -14,14 +14,27 @@ final class GroupCollectionViewCell: UICollectionViewCell {
     // 위 식을 통해 뷰 컨트롤러에 데이터 전달
     var groupData: Group? {
         didSet {
-            guard let groupData = groupData else { return }
-            if let imageString = groupData.groupImage, let image = UIImage(named: imageString) {
+            guard let groupData = groupData else {
+                groupImage.image = UIImage(named: "defaultStoryImage") // 기본 이미지 설정
+                groupNameLabel.text = "---"
+                return
+            }
+
+            // TODO: - 추후 systemName을 named로 변경
+            if let imageString = groupData.groupImage, let image = UIImage(systemName: imageString) {
                 groupImage.image = image
             } else {
```

Comment:
```suggestion
            if let imageString = groupData.groupImage,
               let image = UIImage(systemName: imageString) {
                  groupImage.image = image
            } else {
```
이런 형식으로 작성하면 가독성이 더 좋지 않을까 싶었는데 막상 적고 보니까 아닌 것 같기도 하고... 참고하시라고 작성해봅니다🙄